runtime/src/lib.rs

```diff
@@ -370,6 +379,104 @@ impl pallet_dapps_staking::traits::IsContract for SmartContract {
 	}
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Encode, Decode, MaxEncodedLen)]
+pub struct BondStakeInput<AccountId, Balance> {
+	account_id: AccountId,
+	value: Balance,
+}
+
+/// Contract extension for Local Chain-Extension
+pub struct LocalChainExtension;
+
+impl ChainExtension<Runtime> for LocalChainExtension {
+	fn call<E: Ext>(func_id: u32, env: Environment<E, InitState>) -> Result<RetVal, DispatchError>
+	where
+		<E::T as SysConfig>::AccountId: UncheckedFrom<<E::T as SysConfig>::Hash> + AsRef<[u8]>,
+	{
+		match func_id {
+			// 20 is dapps_staking prefix and 01 is function number
+			// dapps_staking - current_era()
+			2001 => {
```

Comment:
let's start using pallet numbers. dapps-staking is 34. That would make this function to be 3401
https://github.com/AstarNetwork/Astar/blob/88017efb43f838f7f89970366d35c89ade734082/runtime/astar/src/lib.rs#L753

Reply:
Great idea